lib/ASTGen/Sources/ASTGen/Bridge.swift

```diff
@@ -56,6 +44,30 @@ extension String {
   }
 }
 
+extension SyntaxProtocol {
+  /// Obtains the bridged start location of the node in the source buffer provided by `astgen`.
+  ///
+  /// - Parameter astgen: The visitor providing the `ASTContext` and source buffer.
```

Comment:
The AST context is irrelevant to constructing source locations. Could we mention only the source buffer?

Reply:
Thanks, yep!